lib/jekyll-paginate-v2/generator/paginationModel.rb

```diff
@@ -240,6 +241,9 @@ def paginate(template, config, site_title, all_posts, all_tags, all_categories,
           end
 
           using_posts.sort!{ |a,b| Utils.sort_values(Utils.sort_get_post_data(a.data, sort_field), Utils.sort_get_post_data(b.data, sort_field)) }
+          
+          # Remove the first x entries
+          using_posts.pop(config['offset'])
```

Comment:
This needs to make sure that the offset value is not a negative number otherwise we will remove from the end of the list.